src/js/select2/i18n/fa.js

```diff
@@ -0,0 +1,38 @@
+/* jshint -W100 */
```

Comment:
We can use **half-space character** in Farsi language to join some related words together. So if I omit this jshint options, `This character may get silently deleted by one or more browsers.` error will be triggered.

```
Running "jshint:code" (jshint) task

   src/js/select2/i18n/fa.js
     26 |      var message = 'شما تنها می‌توانید ' + args.maximum + ' آیتم را انتخاب نمایید';
                                        ^ This character may get silently deleted by one or more browsers.
     31 |      return 'هیچ نتیجه‌ای یافت نشد';
                               ^ This character may get silently deleted by one or more browsers.

>> 2 errors in 81 files
Warning: Task "jshint:code" failed. Use --force to continue.
```